wp-content/themes/wp-starter/plopfile.js

```diff
@@ -0,0 +1,118 @@
+module.exports = function (plop) {
+
+	// Block icons from https://developer.wordpress.org/resource/dashicons/
+	const WPIcons = [
```

Comment:
It may just be easier to skip this step during the creation process and just default it to the cog or block icon. Supposedly Dashicons have been on their way out for like... a long time, but they're still being used.

Reply:
I kept going back and forth on this. I like it but we are also thinking of maybe doing our own custom blocks which would be cool to put here vs the WP icons. Maybe I will pull this and can add it back in if/when we get custom icons added.